app/models/user_category.rb

```diff
@@ -25,6 +25,13 @@ class UserCategory < ApplicationRecord
   has_many :sub_categories, through: :expense_categories
   has_many :transactions, dependent: :destroy
 
+  def is_Valid?
+    name!="Personal"
```

Comment:
```suggestion
    name != "Personal"
```

Also create a constant for this string.